x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -98,6 +109,96 @@ const tableEntries = [
   },
 ];
 
+const synthtraceHostsTableEntries = [
+  {
+    title: 'host-1',
+    cpuUsage: '90%',
+    normalizedLoad: '18.8%',
+    memoryUsage: '35%',
+    memoryFree: '44.7 GB',
+    diskSpaceUsage: '1,223%',
+    rx: '1.5 Mbit/s',
+    tx: '1.5 Mbit/s',
+  },
+  {
+    title: 'host-2',
+    cpuUsage: '70%',
+    normalizedLoad: '18.8%',
+    memoryUsage: '35%',
+    memoryFree: '44.7 GB',
+    diskSpaceUsage: '1,223%',
+    rx: '1.5 Mbit/s',
+    tx: '1.5 Mbit/s',
+  },
+  {
+    title: 'host-3',
+    cpuUsage: '50%',
+    normalizedLoad: '18.8%',
+    memoryUsage: '35%',
+    memoryFree: '44.7 GB',
+    diskSpaceUsage: '1,223%',
+    rx: '1.5 Mbit/s',
+    tx: '1.5 Mbit/s',
+  },
+  {
+    title: 'host-4',
+    cpuUsage: '40%',
+    normalizedLoad: '18.8%',
+    memoryUsage: '35%',
+    memoryFree: '44.7 GB',
+    diskSpaceUsage: '1,223%',
+    rx: '1.5 Mbit/s',
+    tx: '1.5 Mbit/s',
+  },
+  {
+    title: 'host-5',
+    cpuUsage: '30%',
+    normalizedLoad: '18.8%',
+    memoryUsage: '35%',
+    memoryFree: '44.7 GB',
+    diskSpaceUsage: '1,223%',
+    rx: '1.5 Mbit/s',
+    tx: '1.5 Mbit/s',
+  },
+  {
+    title: 'host-6',
+    cpuUsage: '10%',
+    normalizedLoad: '18.8%',
+    memoryUsage: '35%',
+    memoryFree: '44.7 GB',
+    diskSpaceUsage: '1,223%',
+    rx: '1.5 Mbit/s',
+    tx: '1.5 Mbit/s',
+  },
+];
+
+const SYNTH_HOSTS = [
```

Comment:
(nit)I see you have the same variable defined in the home_page.ts as well.
Since you already have a constants.ts file, it could be a good idea to move it there and then import it.

Unless you need different values for cpuValue, then its fine

Reply:
They have the same values but for example home_page.ts and node_details.ts have the same but for hosts_view.ts we need 6 hosts instead of 5.
My idea here is to have a constant in the test suite that fits each one, might be the same or not, but I want to make it flexible for the needed tests in each one. Making it a shared constant is a good option but any change or addition to it it will have repercussion on each one of the tests. We will probably iterate to this again in the future so we make decide later what's the best.